src/client/client.js

```diff
@@ -49,6 +45,9 @@ const createCashay = (store, cashaySchema) => {
     // ENABLE THIS FOR EXPLORING FRONT END PERFORMANCE
     // const {whyDidYouUpdate} = require('why-did-you-update');
     // whyDidYouUpdate(React);
+    // ENABLE THIS AS ANOTHER MEANS OF TESTING FRONT END PERFORMANCE
+    // const Perf = require('react-addons-perf');
+    // window.Perf = Perf;
```

Comment:
this app is WILDLY unoptimized, it's true. And I know exactly how to speed it up. I have the BEST ways to speed it up, ask anyone. I just haven't worried about that unless I see a screen flicker.